hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/snapshot/TestSnapshotManager.java

```diff
@@ -92,4 +92,15 @@ public void testSnapshotLimits() throws Exception {
           StringUtils.toLowerCase(se.getMessage()).contains("rollover"));
     }
   }
+
+
+  @Test
+  public void testValidateSnapshotIDWidth() {
+    FSDirectory fsdir = mock(FSDirectory.class);
+    SnapshotManager snapshotManager = new SnapshotManager(new Configuration(),
+        fsdir);
+    Assert.assertTrue(snapshotManager.
```

Comment:
can you add a description/comment to explain this assertion.

Reply:
Added!